daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Ref.scala

```diff
@@ -285,4 +285,5 @@ object Ref {
   type TypeSynName = Identifier
   val TypeSynName = Identifier
 
+  case class ModuleId(packageId: PackageId, moduleName: ModuleName)
```

Comment:
I am not very queen on adding this globally for a very limited purpose. 
Please define more locally where you need or use daml.lf.language.Reference.Module

Reply:
@remyhaemmerle-da this is just temporary.

@chunlokling-da let's mark the PR as draft for now

Reply:
Good shout on `daml.lf.language.Reference.Module`. We can use that